doc/nrf/gs_ins_fork.rst

```diff
@@ -3,58 +3,83 @@
 Forking the |NCS| repositories
 ##############################
 
-If you want to change any of the code that is provided by the |NCS| and contribute this updated code back to the |NCS|, you should create your own forks of the |NCS| repositories and clone these forks instead of the base repositories.
+If you want to change any of the code that is provided by the |NCS| and contribute this updated code back to the |NCS|, you should create your own forks of the |NCS| repositories.
 
-To fork and clone the repositories, complete the following steps:
+To fork the repositories and update your cloned repositories to use your fork, complete the following steps:
 
 1. Fork the four |NCS| repositories to your own GitHub account by clicking the **Fork** button in the upper right-hand corner of each repository page:
 
-   a. |ncs_zephyr_repo|
-   #. |ncs_repo|
+   a. |ncs_repo|
+   #. |ncs_zephyr_repo|
    #. |ncs_mcuboot_repo|
    #. |ncs_nrfxlib_repo|
 
-#. Create a folder named ``ncs``.
-   This folder will hold all |NCS| repositories.
 #. Open a GIT bash or terminal window in the ``ncs`` folder.
-#. Clone your forks by entering the following commands, where *<username>* must be replaced with your GitHub user name:
+#. For each of the four repository folders, complete the following steps, replacing *Folder* with the respective folder name (``nrf``, ``zephyr``, ``mcuboot``, or ``nrfxlib``):
 
-   .. code-block:: console
+   a. Enter the respective folder:
 
-      git clone https://github.com/<username>/fw-nrfconnect-zephyr.git zephyr
+      .. parsed-literal::
+         :class: highlight
 
-      git clone https://github.com/<username>/fw-nrfconnect-mcuboot.git mcuboot
+         cd *Folder*
+   #. Add the fork that you created as "default" remote with the name ``origin``.
+      To do so, enter one of the following commands, replacing *Username* with your GitHub user name:
 
-      git clone https://github.com/<username>/fw-nrfconnect-nrf.git nrf
+      For nrf:
+         .. parsed-literal::
+            :class: highlight
 
-      git clone https://github.com/<username>/nrfxlib.git nrfxlib
+            git remote add origin https\://github.com/*Username*/fw-nrfconnect-nrf.git
```

Comment:
For each of those you are missing:
`git remote rename origin ncs`
Since by default, and if you've cloned with west, then the only remote will be called `origin` already

Reply:
OK, I only tested that on one of the other repositories. It is confusing that they behave differently.

Reply:
Yes, it is confusing I agree. This is essentially a consequence of the star topology we have chosen for NCS. The central repository, nrf, does not come from a manifest file and so cannot name its remote anything else than `origin` since at the time there's nothing else known.